elastic/kibana
# [Synthetics] Validate API keys

## Summary

Fixes https://github.com/elastic/kibana/issues/142875

### Testing deletion

1. Enable monitor management, add some monitors 
2. Delete the synthetics api key from stack management view
3. Visit monitor managed and you should see the following error
4. After re-enabling it should work as expected

### Testing permissions

1. Temporarily in the code change the permissions for the API Key
2. After enabling monitor management again you should see the same error

![image](https://user-images.githubusercontent.com/3505601/197560843-83736a59-2a24-4342-8cb2-3f456a98c1ab.png)

Read only user
<img width="1365" alt="Screen Shot 2022-11-14 at 10 39 17 AM" src="https://user-images.githubusercontent.com/11356435/201701938-01d70146-c195-4e4b-abe2-d6a74b2b3777.png">
